RawSpeed/Common.h

```diff
@@ -133,6 +99,19 @@ inline bool isPowerOfTwo (int val) {
   return (val & (~val+1)) == val;
 }
 
+template<typename T> bool isIn(const T value, const std::initializer_list<T>& list) {
+  for (auto t : list)
+    if (t == value)
+      return true;
+  return false;
+}
+
+// until we allow c++14 code
+template<typename T, typename... Args>
+std::unique_ptr<T> make_unique(Args&&... args) {
```

Comment:
Nothing happens. They are in a different namespace. Well, the compiler might still complain because generally both namespaces are included with `using` declarations. I'll just guard it with some `#if`, which is trivial.